docs/scraping-receivers.md

```diff
@@ -0,0 +1,143 @@
+# Scraping metrics receivers
+
+Scraping metrics receivers collect predefined set of metrics from externals sources. Typically, the external source 
+is a monitored entity providing data about itself in some arbitrary format, scraping receivers parse the data and 
+translate it to the OpenTelemetry Pipeline Data. Examples of the scrapings metrics receivers:
+
+- [Host Metrics Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/hostmetricsreceiver)
+- [Redis Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/redisreceiver)
+
+## Defining emitted metrics 
+
+Each scraping receiver has `metadata.yaml` file that MUST define all the metrics emitted by the receiver. The
+file is being used to generate an API for metrics recording, user settings to customize the emitted metrics and user
+documentation. The file schema is defined in 
+https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/cmd/mdatagen/metric-metadata.yaml. 
+Defining a metric in `metadata.yaml` DOES NOT guarantee that the metric will always be produced by the receiver. In
+some cases it may be impossible to fetch particular metrics from a system in a particular state. If it's known that 
+the metric is reported only under certain conditions (e.g. a specific version) of the external source, the 
+condition MUST be mentioned in the metric description.
+
+There are two categories of the metrics emitted by scraping receivers:
+
+- **Default metrics**: emitted by default, but can be disabled in user settings.
+
+- **Optional metrics**: not emitted by default, but can be enabled in user settings.
+
+### How to identify if new metric should be default or optional?
+
+There is no strict rule to differentiate default metrics from optional. As a rule of thumb, default metrics SHOULD be 
+treated as metrics essential to determine healthiness of a particular monitored entity. Optional metrics on the 
+other hand SHOULD be treated as a source of additional information about the monitored entity. 
```

Comment:
Thanks @djaglowski for providing the list. I agree that we should define it as a minimal requirement for marking a metric as default. But I don't agree that we shouldn't even try to consider "usefulness" of a metric before picking up the category. I'm afraid that if we go with this strict set of rules, a user that needs a rare metric which isn't not needed to anyone else will be encouraged to add it as default one, that will make other users to disable more and more metrics in their settings. It can be especially disruptive over time when receiver is marked beta or stable.

Reply:
@djaglowski I incorporated the guidelines that you suggested in the doc. PTAL